scripts/check_setting_visibility.py

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/env python3
+#
+# This script check correctness of settings visibility list
+#
+from typing import Dict
+import os
+import sys
+import json
+import configparser
+import glob
+
+# Directory where this python file resides
+SCRIPT_DIR = os.path.dirname(os.path.realpath(__file__))
```

Comment:
This script is attached to CI, it is triggered every time, @LipuFei made changes for this.
It was implemented outside the UM because we already had the issue in getting file using the path to a certain file and window build crashed. When we made "check_shortcut_keys.py", @LipuFei knows more details why it was like this.